modules/module_utils/ansible_nailgun_cement.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
```

Comment:
I had to put that file in `./module_utils` for ansible to find it, not into `./modules/modules_utils`.
did you do anything different?